evalml/tests/pipeline_tests/test_graphs.py

```diff
@@ -64,7 +64,9 @@ def test_backend_comp_graph(test_component_graph):
             comp.graph()
 
 
-def test_saving_png_file(tmpdir, test_pipeline):
+def test_saving_png_file(tmpdir, test_pipeline, is_using_conda):
+    if is_using_conda:
+        pytest.skip("Skipping saving_png_file if running during conda build process.")
```

Comment:
This line will only be run during the `build_conda_pkg` job. Since it's just `pytest.skip` I think it's ok to merge.